src/lib/useReactDataTable/useReactDataTable.tsx

```diff
@@ -162,6 +178,7 @@ const useReactDataTable = <TData, TFilter extends FilterModel = Record<string, n
     },
 
     enableExpanding: reactTableOptions?.enableExpanding ?? false,
+    enableColumnPinning: reactTableOptions?.enableColumnPinning ?? false,
```

Comment:
Is this required as reactTableOptions will be forwarded then to the table?

Reply:
No, removed it